### ISSUE TITLE
net6.0: UseWasmSharedBuffer=true is not supported on this platform

### ISSUE THREAD

**Issue by catlan** [Tewr/BlazorFileReader#186]
**Describe the bug**
When updating an existing project to net6.0 / ASP Core 6.0.1 I get this error in the console:

```
Unhandled exception rendering component: UseWasmSharedBuffer=true is not supported on this platform: Unable to acquire IJSUnmarshalledRuntime.
System.PlatformNotSupportedException: UseWasmSharedBuffer=true is not supported on this platform: Unable to acquire IJSUnmarshalledRuntime.
   at Tewr.Blazor.FileReader.FileReaderJsInterop.Initialize()
   at Tewr.Blazor.FileReader.FileReaderService..ctor(IJSRuntime jsRuntime, FileReaderServiceOptions options, IServiceProvider serviceProvider)
   at System.Reflection.RuntimeConstructorInfo.InternalInvoke(Object obj, Object[] parameters, Boolean wrapExceptions)
```

Is this a bug or expected for net6.0? 

**Project type**
**'Client-side'/'CSB'** 

**Environment**
 - Browser [e.g. chrome, safari]: both
 - Version of Blazor.FileReader: v3.3.0.21348
 - Version of .net sdk: net6.0 / ASP Core 6.0.1

**Comment by Tewr** [comment]
Clearly unexpected, i didn't see anything like this in my tests. 
IJSUnmarshalledRuntime is not supposed to be used under .net6. 

Is this a debug or release build?

**Comment by Tewr** [comment]
I've read the code, my .net6 test setup is clearly flawed. I'll try to fix this asap

**Comment by catlan** [comment]
Thx for the fast reply! It's a debug build.

**Comment by Tewr** [comment]
fixed in release v3.3.1.21360. Thanks for your help!
